smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -71,20 +73,14 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
         testController.executableModule.getMessageChannel());
     // Mock chrome.smartCardProviderPrivate API.
     mockControl = new goog.testing.MockControl();
-    TestUtils.setUpMockForExtensionApi(
-        mockControl, testController.propertyReplacer);
-    mockControl.$replayAll();
-
-    chromeApiProvider = new ChromeApiProvider(
-        testController.executableModule.getMessageChannel(),
-        pcscReadinessTracker.promise);
+    mockChromeApi =
+        new MockChromeApi(mockControl, testController.propertyReplacer);
   },
 
   'tearDown': async function() {
     try {
       await testController.disposeAsync();
       pcscReadinessTracker.dispose();
-      assertTrue(chromeApiProvider.isDisposed());
```

Comment:
BTW this assert would be still useful now?

Reply:
added back in https://github.com/GoogleChromeLabs/chromeos_smart_card_connector/pull/946